bundles/org.eclipse.jface/src/org/eclipse/jface/resource/ImageDescriptor.java

```diff
@@ -224,6 +224,22 @@ public ImageDescriptor imageDescriptorFromURI(URI uriIconPath) {
 		}
 	}
 
+	/**
+	 * Convenient method to create an ImageDescriptor from an URI
+	 *
+	 * Delegates to ImageDescriptor createFromURL
+	 *
+	 * @param uriIconPath The URI of the image file.
+	 * @return a new image descriptor
+	 *
+	 * @since 3.19
+	 * @deprecated Use {@link #createFromURI(URI)} instead.
+	 */
+	@Deprecated(since = "3.36")
```

Comment:
Mark for removal maybe?

Reply:
Done. No point in keeping the old method when there is a drop-in replacement.